src/LoginServer/Network/Helpers/AppearanceBarrackPcHelper.cs

```diff
@@ -14,43 +15,52 @@
 
 namespace Melia.Login.Network.Helpers
 {
-	public static class CharacterHelper
+	public static class AppearanceBarrackPcHelper
 	{
-		public static void AddCharacter(this Packet packet, Character character)
+		public static void AddAppearanceBarrackPc(this Packet packet, IAppearanceBarrackPc pc)
```

Comment:
One has to say though, "AppearanceBarrackPc" is a mouthful =P"

Reply:
Yes. I don't like that at all. Do you want to use an abbreviation or different name?